webapp/src/main/java/com/box/l10n/mojito/security/ServiceIdentifierParserConfig.java

```diff
@@ -0,0 +1,51 @@
+package com.box.l10n.mojito.security;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ServiceIdentifierParserConfig {
+  @Value("${l10n.spring.security.header.parser.keyValueDelimiter:;}")
+  protected String keyValueDelimiter;
+
+  @Value("${l10n.spring.security.header.parser.valueDelimiter:=}")
+  protected String valueDelimiter;
+
+  @Value("${l10n.spring.security.header.parser.identifierKey:}")
```

Comment:
Is it intended to leave the default empty here?

Reply:
No. Good spot. This was an oversight